README.md

```diff
@@ -31,6 +31,10 @@ The `ksh` executable, helper libraries and man page can be installed with:
 ninja -C build install
 ```
 
+## Latest Builds
```

Comment:
In the expectation additional platforms will be added in the future I'd recommend using the structure and verbiage from https://github.com/fish-shell/fish-shell/. See the "Getting fish" section. Obviously we'll want to name the section "Getting ksh". I'd even go so far as to include the "macOS" and "Windows" subsections with "TBD" or similar text to let people know we want to make it easy to get a bleeding edge ksh on those platforms.